package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "our-groceries-card",
-    "version": "1.4.0",
+    "version": "1.5.10",

```

Comment:
dont change this file/ ill update the version.. keep repo and author the same

Reply:
Reverted this info back.